### ISSUE TITLE
Gradle runner counts and reports time when scan is accepted/rejected

### ISSUE THREAD

**Issue by dniHze** [gradle/gradle#20000]
This is a really hilarious bug. I was scanning my build for potential issues, found them, but forgot that I run one more build with `--scan` option that I forgot about. When. I found a console with the runner, I rejected ToS with the `no` option, after which the runner started counting and interactively reporting all the elapsed time while waiting for input really fast with `100%` completion status. It stopped and finished only after all elapsed time was shown.  

### Expected Behavior
The runner finishes instantly and reports elapsed time without counting.

### Current Behavior
The runner counts elapsed time interactively real fast and finish only after the real elapsed time is reported. 

### Context
Building Kotlin with `--scan` option in interactive console mode.  

### Steps to Reproduce 
1. Run build with `--scan` on Gradle project.
2. Forget about it for a couple of hours or a day.
3. Select the `no` option.
4. Watch Gradle time-travelling through your console. 

### Your Environment
Gradle: 7.4
OS: macOS Monterey, Intel chip
Project: Kotlin multi-modular project.

**Comment by wolfs** [comment]
Thank you for the bug report! I thought the prompt for `--scan` happens very late in the build, when the state is already at 100%. So I am somewhat surprised that the console would count that up again. It is not a priority now, though we might look into this later.
